pmd-java/src/main/resources/category/java/performance.xml

```diff
@@ -790,6 +790,11 @@ public class SimpleTest extends TestCase {
         <description>
 The java.util.Arrays class has a "asList" method that should be used when you want to create a new List from
 an array of objects. It is faster than executing a loop to copy all the elements of the array one by one.
+
+Note that the result of Arrays.asList() is backed by the specified array,
+changes in the returned list will result in the array to be modified.
+For that reason, it is not possible to add new elements to the returned list of Arrays.asList() (UnsupportedOperationException).
+You must use new ArrayList<>(Arrays.asList(...)) if that is inconvenient for you (e.g. because of concurrent access).
```

Comment:
to be able to user `<` and `>`, you should wrap the whole `<description>` contents under a CDATA

Reply:
Thabks, I will fix that.

Reply:
Done.